.github/workflows/InvalidationFlagger.yml

```diff
@@ -20,14 +21,18 @@ jobs:
     - uses: julia-actions/julia-invalidations@v1
       id: invs_master
     
-    - name: Report invalidation counts
+    - name: Print invalidation report
+      id: report
       run: |
         echo "Invalidations on master: ${{ steps.invs_master.outputs.total }} (${{ steps.invs_master.outputs.deps }} via deps)"
         echo "This branch: ${{ steps.invs_pr.outputs.total }} (${{ steps.invs_pr.outputs.deps }} via deps)"
-      shell: bash
-    - name: PR doesn't increase number of invalidations
-      run: |
-        if (( ${{ steps.invs_pr.outputs.total }} > ${{ steps.invs_master.outputs.total }} )); then
-            exit 1
-        fi
-      shell: bash
+      shell: bash  
+      
+    - name: Report invalidations to check
+      uses: dflydev/check-runs-action@v1
+      with:
+        token: ${{ secrets.IAN_PAT }}
```

Comment:
Need to restore this to `secrets.GITHUB_TOKEN`

Hope this is the last patch to make things work